cmd/osm-injector/osm-injector.go

```diff
@@ -193,7 +193,7 @@ func main() {
 		log.Info().Msgf("OSM reconciler enabled for sidecar injector webhook")
 		err = reconciler.NewReconcilerClient(kubeClient, apiServerClient, meshName, stop, reconciler.MutatingWebhookInformerKey)
```

Comment:
Same comment as before, pass a nil apiserver client if injector is not meant to reconcile CRDs.